scripts/kind-e2e/e2e.sh

```diff
@@ -48,9 +50,14 @@ function test_with_e2e_tests {
 
     cd ${DAPPER_SOURCE}/test/e2e
 
+    if [[ "${focus}" != "all" ]]; then
+      GINKGO_FOCUS="-ginkgo.focus \[${focus}\]"
+    fi
+
     go test -v -args -ginkgo.v -ginkgo.randomizeAllSpecs \
         -submariner-namespace $SUBM_NS -dp-context cluster2 -dp-context cluster3 -dp-context cluster1 \
         -ginkgo.noColor -ginkgo.reportPassed \
+        ${GINKGO_FOCUS} \
```

Comment:
If you go with default `.*` then you can just always pass `-ginkgo.focus ${focus}`